### ISSUE TITLE
Influxdb not available for suse repository

### ISSUE THREAD

**Issue by fredmanre** [influxdata/docs.influxdata.com#2059]

**Comment by sanderson** [comment]
@fredmanre Can you clarify. Are you trying to build InfluxDB from source for the SUSE operating system?

**Comment by fredmanre** [comment]
no, i don´t try build influxdb, just that i tried install influxdb from repository, this program(influxdb) not exist or not available in the repo mentioned
